index.js

```diff
@@ -173,7 +189,7 @@ async function unlinkModules() {
 
   const modules = allModules.map(file => {
     const m = file.match(/^\.link-module-alias-(.*)/);
-    return m && m[1];
+    return m && m[1].replace(/--/g, '/');
```

Comment:
Could we make a helper out of this and the above line? Similar to `getModuleAlias`.

Reply:
Good call. See my changes.